contrib/seeds/generate-seeds.py

```diff
@@ -5,11 +5,12 @@
 '''
 Script to generate list of seed nodes for chainparams.cpp.
 
-This script expects two text files in the directory that is passed as an
+This script expects text files in the directory that is passed as an
```

Comment:
fa2e9d9573dacf nit, `s/text files that is passed as an argument:/text files that are passed as arguments:/`

Reply:
That'd be wrong. The directory is passed, not the text files.

Reply:
Oh ok, I misunderstood on reading. Maybe: "expects to find text files in the directory that is passed as an argument:"

Reply:
Leaving doc rewording for a follow-up, which I am happy to review